src/Compilers/CSharp/Test/Emit/PDB/PDBTests.cs

```diff
@@ -5116,5 +5116,69 @@ static void M()
         }
 
         #endregion
+
+
+        [Fact, WorkItem(4370, "https://github.com/dotnet/roslyn/issues/4370")]
+        public void HeadingHiddenSequencePointsPickUpDocumentFromVisibleSequencePoint()
+        {
+            var source = 
+@"#line 1 ""C:\Async.cs""
```

Comment:
I see :) The path is an opaque string to the test, it isn't passed to any IO APIs. It could be an arbitrary string.

If this was a real code then either the path would be translated via the source map (if given) in which case the ```\``` would be normalized according to the given mapping, or it would be written as is (if source map is not given) to the Portable PDB. In any case the debugger (or any other tool reading Portable PDB) is supposed to be able to handle both ```\``` and ```/``` on all platforms.